app/uk/gov/hmrc/nisp/assets/sass/nisp.scss

```diff
@@ -357,9 +359,21 @@ h1.title-spa {
   padding-bottom: 0px;
 }
 
-table.cope-table p{   
+table.cope-table td {
```

Comment:
as discussed,I am raising ticket and we will do it as tech debt due to urgency of Prod ticket.